samples/ImplicitFlow/AuthorizationServer/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:12345",
+      "applicationUrl": "http://localhost:5000",
```

Comment:
`5000` is so common that it interfered with other apps that might also be running.